about-me.md

```diff
@@ -1,9 +1,9 @@
 # About Me
 
-![Picture of me]()
+![Picture of me](https://avatars0.githubusercontent.com/u/51712006?s=460&v=4)
 
-Hi, I'm ____________. I'm a ________ at *__________*.
+Hi, I'm Ange. I'm a talent expert at *USDS HQ*.
 
-- **Past Companies and Organizations:**
-- **Hobbies:**
-- **Favorite Website:** []()
+- **Past Companies and Organizations:** Telarix/Tomia, EAB, The Advisory Board Company
+- **Hobbies:** Swimming, reading, netflix, baking 
```

Comment:
Please tell me you are a Black Mirror fan.

Reply:
I watch Black Mirror but only very specific episodes (San Junipero, the new Ashley O one) so if you have a recommendation, LMK!